index.d.ts

```diff
@@ -34,21 +35,5 @@ export interface IDraggableProps {
     maxY?: number;
 }
 declare function Draggable(props: IDraggableProps): JSX.Element;
-Draggable.defaultProps = {
-    renderText: '＋',
-    renderSize: 36,
-    shouldReverse: false,
-    disabled: false,
-    debug: false,
-    onDrag: () => {},
-    onShortPressRelease: () => {},
-    onDragRelease: () => {},
-    onLongPress: () => {},
-    onPressIn: () => {},
-    onPressOut: () => {},
-    onRelease: () => {},
-    x: 0,
-    y: 0,
-    z: 1,
-}
+
```

Comment:
I'm not sure how typescript works with defaultProps, might still need these here for typescript intellisense?